WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -472,6 +483,8 @@ public static void WordPressComSignOut(Context context) {
         if (!AccountHelper.isSignedIn()) {
             EventBus.getDefault().post(new UserSignedOutCompletely());
         }
+
+        AnalyticsTracker.track(Stat.ACCOUNT_LOGOUT);
```

Comment:
Should this be called before `removeWpComUserRelatedData()` above, since that clears the user's wp.com info? Just wondering if any of that info is used by analytics.